src/configuration/converters.ts

```diff
@@ -33,6 +36,29 @@ function mapToFareProductTypeConfig(
   return typeConfigPotential.data;
 }
 
+export const mapToFlexibleTransportOption = (
+  filter: any,
+): FlexibleTransportOptionType | undefined => {
+  const safeParseReturnObject =
```

Comment:
I got an uncaught error regarding this when I tried to run the app locally, but I'm not sure I have set things up correctly. It says "Cannot read property safeParse of undefined"

Reply:
Hi, do a yarn install again because it needs an update, we can take a look if still fails